modules/mcagar/src/main/ts/ephox/mcagar/alien/EditorTypes.ts

```diff
@@ -13,14 +13,22 @@ export interface Selection {
 export type ContentFormat = 'raw' | 'text' | 'html' | 'tree';
 
 export interface GetContentArgs {
-  format?: ContentFormat;
-  get?: boolean;
-  content?: string;
+  format: ContentFormat;
```

Comment:
The tests seems to all pass, and since this is mainly just to make the epic be able to move forward while a proper fix is implemented, it should probably be fine?